.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Why is this being set to false?

Reply:
I'm not sure there's any other use for trailing spaces in markdown file. So set the `trim_trailing_whitespace` to default set.